File: CS/FileTree/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using FileTreeList;

namespace FileTree
{
    public partial class Form1 : XtraForm
    {
        private TreeListFileHelper _helper;
        public Form1()
        {
            InitializeComponent();
            _helper = new TreeListFileHelper(treeList1);
        }
    }
}

File: VB/FileTree/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports FileTreeList

Namespace FileTree
	Partial Public Class Form1
		Inherits XtraForm
		Private _helper As TreeListFileHelper
		Public Sub New()
			InitializeComponent()
			_helper = New TreeListFileHelper(treeList1)
		End Sub
	End Class
End Namespace
